core/src/main/java/org/apache/accumulo/core/client/rfile/RFileScanner.java

```diff
@@ -406,8 +406,8 @@ public void close() {
       if (this.blockCacheManager != null) {
         this.blockCacheManager.stop();
       }
-    } catch (RuntimeException e1) {
-      throw new RuntimeException(e1);
+    } catch (UnsupportedOperationException e1) {
+      throw new UnsupportedOperationException(e1);
```

Comment:
I'm confused by this. Why not just let e1 get thrown all the way through?

Reply:
Is this what you meant? 51afbce

Reply:
Or just don't catch it in the first place.